step.yml

```diff
@@ -53,3 +53,16 @@ inputs:
       value_options:
         - "yes"
         - "no"
+  - cache_level: "all"
+    opts:
+      category: Options
+      title: Set the level of cache
+      is_required: true
+      description: |-
+        Sets the level of cache.
+
+        'all' enables the caching of all files under /var/cache/apt/archives folder.
```

Comment:
this is a technical information, please provide a user friendly description of this feature

Reply:
Added bit more details.